app/api-v1/services/recipeService.js

```diff
@@ -6,9 +6,9 @@ async function createRecipe(reqBody) {
     throw new BadRequestError({ message: 'Invalid recipe input' })
   }
 
-  const { imageAttachmentId } = reqBody
+  const { image_attachment_id } = reqBody
```

Comment:
you can't just update these. They are part of the API doc

Reply:
this is where it gets mapped -> https://github.com/digicatapult/inteli-api/blob/main/app/api-v1/routes/recipe.js#L37-L46